app/build.gradle

```diff
@@ -99,7 +99,11 @@ dependencies {
     implementation 'com.twitter.sdk.android:twitter-core:3.1.1'
     implementation 'com.twitter.sdk.android:tweet-ui:3.1.1'
 
+    // Facebook APIs
+    implementation 'com.facebook.android:facebook-android-sdk:[4,5)'
```

Comment:
Only for curiosity what is this syntax `facebook-android-sdk:[4,5)` ? What does `[4, 5)` mean ?

Reply:
I really don't know, just I needed this to use facebook

Reply:
ok :)